### PR TITLE
Draft of glossary file

### PR DESCRIPTION
**Pull request by kmsholcomb**
I created a draft of a glossary for our container content that we could surface for customers. It needs to be reviewed, and definitions need to be added and changed. Also, I am not sure about the placement. A glossary is a reference-type topic, but I am not sure that we would want it in the "references" folder.

### CODE REVIEW

**Comment by kmsholcomb** [comment]
Related to https://github.com/rackerlabs/docs-container-service/issues/57

File: _references/glossary.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+A pattern for linking containers. The ambassador pattern uses Docker links with specialized ambassador containers to enable communication between containers across Docker hosts. 
+
+##### Carina
+
+[add a definition]
+
+##### cluster
+
+[add a definition]
```
Comment (kmsholcomb) [inline _references/glossary.md]: A pool of compute, storage, and networking resources that serves as a host for one or more containerized applications.

(Definition from Everett's "zero-to-hero" article)

**Comment by kmsholcomb** [comment]
Added definition for cluster, and hid entry for Carina until I get a good definition.

File: _references/glossary.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+A tool used to define and run multicontainer applications with Docker. With Compose, you use a YAML file to define a multicontainer application, and then start the application with a single command.
+
+##### Docker daemon	
+
+Runs in a Docker host and builds, runs, and distributes containers. Users interact with the daemon through the Docker *client*. Sometimes referred to as the Docker *server* or *engine*.  	
+
+##### Docker host
+
+Runs in a virtual machine built by the *Docker Machine*. The host contains the daemon, and it is where images are built and containers are created. In Carina, the equivalent of a Docker host is a *segment*.
```
Comment (carolynvs) [inline _references/glossary.md]: It isn't necessarily run in a VM, that's only when using Docker locally on Windows or Mac. Linux is the only operating system which can currently host the Docker service/engine.

File: _references/glossary.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+The public Docker *registry* from which you can upload or download images.
+ 
+##### Docker image
+
+A read-only template that is used to create Docker containers. For example, an image could contain an Ubuntu operating system with Apache and your web application installed. You can download public images or create your own. Images are stored in *registries*.   	
+
+##### Docker links
+
+A pattern for linking containers that enables containers that are on the same host to communicate.
```
Comment (carolynvs) [inline _references/glossary.md]: May want to change it to something like "A Docker configuration option which connects containers that enables..."

**Comment by carolynvs** [comment]
:+1:

File: _references/glossary.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+The following terms are used in the documentation for Carina.
+
+##### ambassador pattern
+
+A pattern for linking containers. The ambassador pattern uses Docker links with specialized ambassador containers to enable communication between containers across Docker hosts. 
+
+##### AppArmor profile
+
+A security layer in Carina that keeps conatiner resources isolated. It denies access to sensitive file paths on the hosts, whitelists expected mount calls for the container file system, and restricts the device capabilities of the Docker process.
```
Comment (annegentle) [inline _references/glossary.md]: conatiner > container

Comment (annegentle) [inline _references/glossary.md]: AppArmor is a Linux kernel security module and the AppArmor profile is a security layer in Carina keeps container resources isolated. The profile denies access...

File: _references/glossary.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+##### Carina
+
+A hosted container runtime environment provided by Rackspace. Carina is built on the open-source Docker Swarm project and provides bare-metal speeds, security, and portability for running yor applications in containers.
```
Comment (annegentle) [inline _references/glossary.md]: your

File: _references/glossary.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+A hosted container runtime environment provided by Rackspace. Carina is built on the open-source Docker Swarm project and provides bare-metal speeds, security, and portability for running yor applications in containers. 
+
+##### cluster
+
+A pool of compute, storage, and networking resources that serves as a host for one or more containerized applications. Carina provisions Docker Swarm clusters for running containers.
+
+##### container
+
+An userspace instance that packages a piece of software with a complete file system that contains everything that the software needs to run, such as code, system tools, and system libraries. Each container is an isolated and secure application platform. A container is created from an *image*. Containers can be run, started, stopped, moved, and deleted.
```
Comment (annegentle) [inline _references/glossary.md]: An userspace? Looks funny to me but what do I know?

File: _references/glossary.md (hunk 5 above)
Comment (kmsholcomb) [inline _references/glossary.md]: Thanks for finding the typos!!

File: _references/glossary.md (hunk 6 above)
Comment (kmsholcomb) [inline _references/glossary.md]: You know better than to use "an" there!

**Pull request merged**